apps/wing/src/analytics/scripts/detached-export.ts

```diff
@@ -0,0 +1,58 @@
+import { AnalyticsStorage } from '../storage';
+import * as fs from 'fs';
+import Analytics from '@segment/analytics-node';
+import { exit } from 'process';
+
+// When this file is run as a child process, it will be passed the path 
+// to the analytics report file
+const filePath = process.argv[2];
+
+async function reportAnalytic() {
+  if (process.env.DEBUG) {
+    // In debug mode no need to export the metrics
+    exit(0);
+  }
+
+  if (!filePath) {
+    throw new Error('No file analytic path provided');
+  }
+
+  const analytics = new Analytics({ writeKey: 'sCqPF5xSscOjJdi5Tbkqu73vfF8zkZdw'});
+  const storage = new AnalyticsStorage();
+  const event = storage.loadEvent(filePath);
+
+  if (!event) {
+    throw new Error(`No analytic event found at: ${filePath}`);
+  }
+
+  const params = {
+    anonymousId: (event.properties as any).ci_name ? 'ci_run' : storage.getAnonymousId(),
```

Comment:
`ci_run` is not a good value. But because I have `anonymous_repo_id` I can work the details in the dashboards. Lets revert to using `getAnonymousId`

```suggestion
    anonymousId: storage.getAnonymousId(),
```